app/src/main/java/org/jellyfin/androidtv/util/Utils.java

```diff
@@ -1635,6 +1628,7 @@ public void onResponse(ArrayList<ServerInfo> serverResponse) {
                     }
                 });
                 break;
```

Comment:
How is it possible to get in this state if you're removing connect login?

Reply:
That state is set by the Java API client when no servers are available. That logic should probably be removed at some point.

https://github.com/jellyfin/jellyfin-apiclient-java/blob/master/android/src/main/java/mediabrowser/apiinteraction/connectionmanager/ConnectionManager.java#L133

Reply:
Thanks!